### ISSUE TITLE
Compose generated string resources does not appear on testflight installed apps

### ISSUE THREAD

**Issue by Mats-Hjalmar** [JetBrains/compose-multiplatform#4736]
**Describe the bug**
If you use a string compose resource it does not work in a testflight downloaded version of you app.

**Affected platforms**
- iOS

**Versions**
- Kotlin version*:  1.9.23
- Compose Multiplatform version*: 1.6.10-dev1608
- OS version(s)* (required for Desktop and iOS issues):  17.4
- OS architecture (x86 or arm64): arm64
- Device (model or simulator for iOS issues): IPhone 14 Pro, IPhone 13 Pro

**To Reproduce**
Add a string to en/strings.xml resource folder.
Use this string in in an iOS targeted app.
Upload this app to testflight.
Run it.

**Expected behavior**
The resource should be found

**Additional context**
Once downloaded from testflight i cant install the app locally without clearing caches.

**Comment by igordmn** [comment]
> i cant install the app locally without clearing caches.

Does it mean that the issue reproduces only when you try to update the application? If so, then from which Compose version upgrade happens? Does it reproduce with clean install?

**Comment by Mats-Hjalmar** [comment]
No, it does not only reproduce on update. It reproduces on clean install.
1. Deleting the app and installing via xcode is still broken (even if this same build is working on a simluator (or any unafected device).
2. Removing this string resource -> Upload to testflight -> download it to the same phone -> Works.

**Comment by igordmn** [comment]
We didn't able reproduce the issue - an app installed via TestFlight can access resources successfully.

We would appreciate a sample project with the issue, and the detailed steps how you build and upload the application.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-4736">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
